python/dlr/counter/counter_mgr.py

```diff
@@ -0,0 +1,153 @@
+                        feature_enb = False
+                    else:
+                        feature_enb = True
+            else:
+                feature_enb = True
+        except Exception as e:
+            logging.exception("while in reading ccm config file")
+        return feature_enb 
+
+    def is_device_info_published(self):
+        """check if device information send only single time in DLR installation life time.
+            create a dummy binary file in DLR installation path to record process state.
+        """
+        flag = False
+        try:
+            ccm_rec_data_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                             config.CALL_HOME_RECORD_FILE)
+            if os.path.exists(ccm_rec_data_path):
+                flag = True
+            else:
+                # write runtime_load as a flag in a file
+                with open(ccm_rec_data_path, "wb") as fp:
+                    fp.write(CallCounterMgr.RUNTIME_LOAD.to_bytes(1, byteorder='big'))
+        except IOError as e:
+            logging.exception("while reading ccm publish data record file")
+        except Exception as e:
+            logging.exception("while reading ccm publish data record file")
+        return flag
+
+    def runtime_loaded(self):
+        """push device information on AI.DLR library load"""
+        try:
+            if not self.is_device_info_published():
+                pub_data = {'record_type': CallCounterMgr.RUNTIME_LOAD}
+                if self.system:
+                    pub_data.update(self.system.get_device_info())
+                self.push(pub_data)
+        except Exception as e:
+            logging.exception("while dlr runtime load", exc_info=True)
+
+    def model_loaded(self, model):
+        self.model_info_publish(CallCounterMgr.MODEL_LOAD, model)
+
+    def model_run(self, model):
+        _md5model = get_hash_string(model.encode())
+        _md5model = str(_md5model.hexdigest())
+        ModelExecCounter.update_model_run_count(_md5model)
+
+    def model_info_publish(self, model_event_type, model, count=0):
+        """push model load information at time model load time"""
+        try:
+            _md5model = get_hash_string(model.encode())
+            _md5model = str(_md5model.hexdigest())
+            pub_data = {'record_type': model_event_type, 'model': _md5model}
+            if self.system:
+                pub_data['uuid'] = self.system.get_device_uuid()
+            if model_event_type == CallCounterMgr.MODEL_RUN:
+                pub_data['run_count'] = count
+            self.push(pub_data)
+        except Exception as e:
+            logging.exception("unable to complete model count", exc_info=True)
+
+    def push(self, data):
+        """publish information to Server"""
+        if self.msg_publisher:
+            self.msg_publisher.send(json.dumps(data))
+
+    def stop(self):
+        with relinst:
+            if self.model_metric:
```

Comment:
There is a null pointer exception here when `if self.system is None:` in line 64.

Please fix this and add unit test for unsupported system as well